doc/source/installation.rst

```diff
@@ -97,6 +97,30 @@ Here's a summary of the variations:
 
 .. _ray-install-java:
 
+Installing a specific dependency variant
+----------------------------------------
+To get finer grained control over the size of your dependencies for your Ray application, Ray provides a few optional dependency installation specifications. 
+
+These variants can be installed with:
+
+.. code-block:: bash
+
+    pip install ray[variant_string_goes_here]
+
+Availiable variant strings include:
+
+* ``default``: same as ``pip install ray``, includes things like :ref:`Ray dashboard <ray-dashboard>` and cluster launcher
```

Comment:
A couple of comments:
- Can we say specifically which things it includes rather than "things like"?
- For me it's not clear if the others also include autoscaler, CLI, etc.
- Is it "same as **but** includes" or "same as **i.e.** includes"? My understanding was autoscaler, CLI, etc. are not included with ``pip install ray`` but are included with ``pip install ray[default]`` but the current wording reads like they both are identical.

Reply:
@amogkam what things do all share? do they all share autoscaler? CLI?

Reply:
also we should spell out difference between `ray` and `ray[default]` more clearly. do you have a blurb for that?